code/datums/gamemodes/game_mode.dm

```diff
@@ -303,6 +303,10 @@ Sensors indicate [numXenosShip || "no"] unknown lifeform signature[numXenosShip
 		addtimer(CALLBACK(GLOBAL_PROC, .proc/send_global_signal, COMSIG_GLOB_OPEN_TIMED_SHUTTERS_LATE), SSticker.round_start_time + 30 MINUTES)
 			//Called late because there used to be shutters opened earlier. To re-add them just copy the logic.
 
+	if(flags_round_type & MODE_LZ_SHUTTERS)
```

Comment:
add another flag for it. MODE_SURVIVOR or whatever, and give it to hunt and distress

Reply:
will do after i fix da linter